acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonNodeFactoryUtils.java

```diff
@@ -49,7 +49,7 @@
         validators, template, CliqueExtraData::createGenesisExtraDataString);
   }
 
-  public Optional<String> createIbftGenesisConfig(
+  public Optional<String> createIbftTwoGenesisConfig(
       final Collection<? extends RunnableNode> validators) {
     final String template = readGenesisFile("/ibft/ibft.json");
```

Comment:
This genesis file should be renamed as well

Reply:
done